packages/mock/src/client.ts

```diff
@@ -745,10 +756,31 @@ export class MockFetchClient {
 
     const result = await this.router.handleRequest(request, this.repo);
     if (result.length === 1) {
+      const issueDetails = result[0].issue[0]?.details;
```

Comment:
I've seen destructuring used in several other places in the code to clarify like this:
```suggestion
    const [outcome, resource] = await this.router.handleRequest(request, this.repo);
    if (!resource) {
      const issueDetails = outcome.issue[0]?.details;
```